modules/tinymce/src/core/test/ts/browser/html/DomParserTest.ts

```diff
@@ -530,63 +530,6 @@ describe('browser.tinymce.core.html.DomParserTest', () => {
         );
       });
 
-      it('Remove redundant br elements', () => {
```

Comment:
Still keep these tests until the filter & option are actually removed from `DomParser`.

Reply:
revered change